gremlin-server/pom.xml

```diff
@@ -63,10 +63,12 @@ limitations under the License.
         <dependency>
             <groupId>com.codahale.metrics</groupId>
             <artifactId>metrics-graphite</artifactId>
+            <optional>true</optional>
```

Comment:
All of metrics is under `<dependencyManagement>` in the root pom.xml so the version is defined there:

https://github.com/apache/tinkerpop/blob/268333b4778c87ce6c2429eda97e3e1adc7757e3/pom.xml#L602-L642

I don't think it needs to be specified again.